make_group_file.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
+
+"""
+This script will
+
+- open genotype file
+- open gene cis window files
+- create gene rv group files
+
+these files will be used as inputs for the
+SAIGE-QTL association pipeline.
+
+To run:
+
+analysis-runner \
+    --description "make variant group input files" \
+    --dataset "bioheart" \
+    --access-level "test" \
+    --output-dir "saige-qtl/input_files/" \
+    python3 make_group_file.py --chromosomes chr22 \
+        --cis-window-files-path saige-qtl/input_files/ \
+        --group-file-path gs://cpg-bioheart-test/saige-qtl/input_files/group_files/
+
+
+"""
+
+import click
+import logging
+
+import hail as hl
+import pandas as pd
+
+import hailtop.batch.job as hb_job
+from typing import List
+
+from cpg_utils import to_path
+from cpg_utils.config import get_config
+
+from cpg_utils.hail_batch import dataset_path, get_batch, init_batch
+
+
+def distance_to_weight(distance: int, gamma: float = 1e-5):
```

Comment:
You shouldn't really be importing inside the method definition, which only leaves one functional line in this method. You should probably just do the `math.exp(-gamma * abs(distance))` inside your list comprehension instead of having a method here

Reply:
ok!